src/understand-tidb/privilege-management.md

```diff
@@ -1,2 +1,65 @@
 # Privilege management
 
+At its core, TiDB's approach to user privileges is similar to that of MySQL:
+- The privileges are stored in tables such as `mysql.user` and `mysql.db`.
+- The privilege tables are then loaded into an [in-memory cache](https://github.com/pingcap/tidb/blob/master/privilege/privileges/cache.go). The cache is then used by the privilege manager to determine the privileges of a user.
+- The cache is automatically updated when using privilege control statements such as `GRANT` and `REVOKE`. The statement `FLUSH PRIVILEGES` can also be used to manually reload the cache for when manual changes are made to the privilege tables.
+
+## Behavior differences from MySQL
+
+Implicit updates to the privilege cache (i.e. when `GRANT` or `REVOKE` statements are executed) run immediately on the instance of TiDB that is executing the statement. A [notification is also sent to all TiDB instances](https://github.com/pingcap/tidb/blob/5e05922de6a253859cfbfe19356de8a2e2db39da/domain/domain.go#L1355-L1373) to rebuild their cache. This notification is sent asynchronously, so it is possible that when a load balancer is used, the cache will be out of date when attempting to reconnect to a TiDB instance immediately.
+
+Because the asynchronous notifications do not guarantee delivery, TiDB will also [automatically rebuild the privilege cache](https://github.com/pingcap/tidb/blob/5e05922de6a253859cfbfe19356de8a2e2db39da/domain/domain.go#L852-L908) every 5-10 minutes in a loop. This behavior is not strictly MySQL compatible, because in MySQL the privilege cache will only ever be rebuilt from a `FLUSH PRIVILEGES` statement, a restart, or a privilege control statement.
+
```

Comment:
Do you think the difference in `information_schema` table should be included? For example, TiDB adds a new `global_priv` table that does not exist in MySQL. (I'm not sure)

Reply:
could you add a link to user privileges of MySQL here? it helps cross check for whom is not so familiar with this part

Reply:
sorry, it is already added at the end